lib/presentation/mixins/common_media_picker_mixin.dart

```diff
@@ -80,14 +82,36 @@ mixin CommonMediaPickerMixin {
       context,
       pickerConfig: onlyImage
           ? CameraPickerConfig(
+              textDelegate: getTextDelegateForLocale(
+                context,
+              ),
               enableAudio: false,
               onError: (e, a) => _onError(context: context, error: e),
             )
           : CameraPickerConfig(
+              textDelegate: getTextDelegateForLocale(
+                context,
+              ),
               enableRecording: true,
               onError: (e, a) => _onError(context: context, error: e),
             ),
-      locale: View.of(context).platformDispatcher.locale,
     );
   }
+
+  CameraPickerTextDelegate getTextDelegateForLocale(
+    BuildContext context,
+  ) {
+    switch (LocalizationService.currentLocale.value.languageCode) {
+      case 'ru':
+      case 'fr':
+        return LocalizedCameraPickerTextDelegate(
+          context,
+          LocalizationService.currentLocale.value.languageCode,
+        );
+      default:
+        return cameraPickerTextDelegateFromLocale(
+          LocalizationService.currentLocale.value,
```

Comment:
it is a huge risk if this locale value is not translated. IMO, fallback case must be `en`

Reply:
When the device is set to an unsupported language, `LocalizationService.currentLocale` defaults to English. As a result, camera_picker strings will also be displayed in English

## Demo with device set to the spanish language

[1817_fallback.webm](https://github.com/user-attachments/assets/f1000e73-9241-4691-9640-239d5402e2a8)